server/odc-core/src/main/java/com/oceanbase/odc/core/sql/split/SqlCommentProcessor.java

```diff
@@ -599,4 +609,71 @@ public int getValue() {
         }
     }
 
+    public class SqlIterator implements Iterator<String>, AutoCloseable {
```

Comment:
further more, the name of this class may be `SqlLine`? user will get a line from every iteration.

Reply:
done. Renamed to `SqlLine`